### ISSUE TITLE
Не работает постоянная раскладка для rasphone.exe

### ISSUE THREAD

**Issue by c1tyguide** [BladeMight/Mahou#395]
Приветствую, такая проблема:
Последняя версия программы, в Win11 x64
не  срабатывает установка постоянной раскладки для окна дозвона rasphone.exe
При этом для notepad.exe все срабатывает как положено.

**Comment by BladeMight** [comment]
> окна дозвона rasphone.exe

Может оно запускается от администратора? Если так тогда Mahou также нужно запускать от администратора.

**Comment by c1tyguide** [comment]
> > окна дозвона rasphone.exe
> 
> Может оно запускается от администратора? Если так тогда Mahou также нужно запускать от администратора.

Нет, не от администратора, на всякий случай стартовал Mahou админом, и то же самое.
А вы у себя можете воспроизвести? Работает?

**Comment by BladeMight** [comment]
> можете воспроизвести

У меня не Win11, поэтому не могу точно сказать. У меня при включении постоянной раскладки 1 для `rasphone.exe` при переключении на окна этого процесса, раскладка меняется на 1, для проверки менял окна с `notepad.exe` перед переключением на окна `rasphone.exe` менял раскладку.

**Comment by c1tyguide** [comment]
Попробовал на рабочем компе тоже с 11 Виндой. Там работает. Но там версия Mahou 2.9.0.1, а на своем я обновил до бэты. Скопировал Mahou с рабочего на свой и вот теперь все в порядке, переключает.

**Comment by BladeMight** [comment]
> версия Mahou 2.9.0.1

Хм. У меня и на latest-commit работает, кол. изменений с 2.9.0.1 огромное - поэтому найти причину непросто. Если тебе не сильно нужны все новые введение в latest-commit можешь пользоваться 2.9.0.1.

Еще как вариант можно попробовать включить: `Эмулировать переключение раскладки.` и поставить в списке `Win+Space`. Если так работает, значит проблема скорее всего в изменении: 214ccbb, вот для проверки еще версия Mahou до этого изменения:
[Mahou-f47d28.zip](https://github.com/BladeMight/Mahou/files/12207900/Mahou-f47d28.zip)

**Comment by c1tyguide** [comment]
Да, спасибо, эта работает нормально. С последней финальной версией у меня еще были проблемы при запуске и выходе из сна, ничего не переключалось если не убить процесс и запустить вручную. Понаблюдаю как с этой будет.
